integration-tests/src/test/java/com/pip/chatbot/integration/tests/tests/weatherAdmin.feature

```diff
@@ -0,0 +1,107 @@
+Feature: Restaurants Api
+
+  Background:
+    * url baseUrl
+    * def initDatabase =
+    """
+    function() {
+      var DbUtils = Java.type('com.pip.chatbot.integration.tests.WeatherDbUtils');
+      var db = new DbUtils(dbConfig);
+      db.initWeatherData();
+     return db;
+    }
+    """
+    * def db = callonce initDatabase
```

Comment:
Zmienne bez przedrostka i wyrażające co jest w środku(citiesUpdate ma cityForUpdate i cityForDelete, to średnio intuicyjne jest)